ghost/core/core/server/data/migrations/versions/5.15/2022-09-14-12-46-add-email-track-clicks-setting.js

```diff
@@ -1,8 +1,8 @@
 const {addSetting} = require('../../utils');
 
 module.exports = addSetting({
-    key: 'email_track_clicks',
```

Comment:
Is there a reason you're changing the old migration? 😅

Reply:
Other than the fact my 🧠 is a mess, no other reason! 😅  Updated now